vue/components/ui/molecules/table/table.vue

```diff
@@ -1,7 +1,27 @@
 <template>
-    <table class="table" v-bind:class="classes" v-bind:style="style">
+    <table
+        class="table"
+        v-bind:class="{ classes, disableSelection: lastIndexWithShift !== null }"
+        v-bind:style="style"
+    >
+        <global-events
+            v-on:keydown.meta.65.exact.prevent="onCtrlA()"
+            v-on:keydown.ctrl.65.exact.prevent="onCtrlA()"
+            v-on:keydown.shift.up.exact="onShiftUp()"

```

Comment:
Still doesn't work, specially when clicking with a mouse in one and then SHIFT+UP/SHIFT+DOWN. The behaviour is weird when nothing is selected.